openmdao_book/features/core_features/working_with_groups/parallel_group.ipynb

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
<p>I will modify this to run in MPI on my branch.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>